lib/components/scripts/writeCssScript.js

```diff
@@ -8,7 +8,7 @@ const fs = require('fs');
 const { parse } = require('css');
 const { isNaN } = require('global');
 
-const INPUT = require.resolve('overlayscrollbars/css/OverlayScrollbars.min.css');
```

Comment:
This is a part of the whole nx thing, but I can not find anywhere where this file is referenced other than inside node_modules - which is weird....